### PR TITLE
Add predicates for long refinements

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: refined/src/test/scala/main/lucuma/RefinedSuite.scala [hunk 1]
```diff
@@ -43,6 +43,16 @@ class RefinedSuite extends FunSuite {
     assertRefineError("2.refined[Less[1]]")
   }
 
+  test("greater long") {
+    2L.refined[Greater[1L]]
+    assertRefineError("1.refined[Greater[2]]")
```
Comment (armanbilge) [inline refined/src/test/scala/main/lucuma/RefinedSuite.scala:48]:
```suggestion
    assertRefineError("1L.refined[Greater[2L]]")
```

File: refined/src/test/scala/main/lucuma/RefinedSuite.scala [hunk 2]
```diff
@@ -43,6 +43,16 @@ class RefinedSuite extends FunSuite {
...
+  }
+
+  test("less long") {
+    1L.refined[Less[2L]]
+    assertRefineError("2.refined[Less[1]]")
```
Comment (armanbilge) [inline refined/src/test/scala/main/lucuma/RefinedSuite.scala:53]:
```suggestion
    assertRefineError("2L.refined[Less[1L]]")
```

**Comment by cquiroz** [comment]
Should we give this a proper version number? lett's say a 0.1 release?

**Comment by armanbilge** [comment]
Sure, sounds good to me :)

**Pull request merged**